code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -822,13 +822,20 @@
 	return FALSE
 
 /mob/living/silicon/robot/updateicon()
-	cut_overlays()
+	overlays.Cut() // The new way the overlays are applied requires this overlays.Cut difference to remove
+// correctly
+
+//The below lines up to the panels are how the 'borgs get their overlays now. It's made cleaner
+// via the vars below.
+
+	var/overlay_help = image(icon,"eyes-[module_sprites[icontype]]-help", EFFECTS_ABOVE_LIGHTING_LAYER)
+	var/overlay_harm = image(icon,"eyes-[module_sprites[icontype]]-harm", EFFECTS_ABOVE_LIGHTING_LAYER)
 
 	if(stat == CONSCIOUS)
 		if(a_intent == I_HELP)
-			add_overlay("eyes-[module_sprites[icontype]]-help")
+			overlays += overlay_help
 		else
-			add_overlay("eyes-[module_sprites[icontype]]-harm")
+			overlays += overlay_harm
```

Comment:
You should also use add_overlay here anyway.

Reply:
As said on Discord, I tried that at first, but I don't know if you can actually apply a layering onto the 'add_overlay' template since it's automatically on the float layer, which puts it just one layer above whatever layer the mob is on.

Also I read somewhere tho I can't find it, that the overlays += function makes overlays not linger around ambiantly after being used or something like that, which was said to be a p. bad overlay bug.

Reply:
Moving as resolved since Matt said it was okay (But re-open this if you change your mind and I'll try to get add_overlay to work again)